dashboard-for-web/articles/web-dashboard-designer-mode/provide-data/manage-data-sources.md

```diff
@@ -21,7 +21,7 @@ The window allows you to do the following:
 | Action | Description |
 |----------|----------|
 | **Add an existing data source** | Select the data source and click the **Add** button.  |
-| **Create a new data source** | Click **Create data source...**. It invokes the [Dashboard Data Source Wizard](dashboard-data-source-wizard.md) where you can create a new data source based on the predefined data connection. Then click **Add** to add the newly created data source to the dashboard data sources.  |
+| **Create a new data source** | Click **Create data source...**. It invokes the [Dashboard Data Source Wizard](../ui-elements/dialogs-and-wizards/dashboard-data-source-wizard.md) where you can create a new data source based on the predefined data connection. Then click **Add** to add the newly created data source to the dashboard data sources.  |
```

Comment:
...`to add the newly created data source to the dashboard data sources.`

Are you adding the data source to a collection of data sources? For example, do you mean:

`to add the newly created data source to the dashboard data source collection.`

Or is it a list of data sources?

`to add the newly created data source to the list of dashboard data sources.`